__tests__/app.test.js

```diff
@@ -41,4 +41,47 @@ describe('1. GET api topics', () => {
                 })
         })
     })
+})
+
+describe('2. GET api articles', () => {
+    describe('a. status 200 & Data', () => {
+        it('returns 200 status code & an array back to the user where the data is in descending order by created_at and a comment count', () => {
+            return request(app)
+                .get('/api/articles')
+                .expect(200)
+                .then( (res) => {
+                    const articles = res.body;
+                    expect(articles).toBeInstanceOf(Array);
+                    expect(articles.length).toBe(12);
+                    articles.forEach(article => {
+                        expect(article).toEqual(
+                            expect.objectContaining({
+                                article_id : expect.any(Number),
+                                title : expect.any(String),
+                                topic : expect.any(String),
+                                author : expect.any(String),
+                                body : expect.any(String),
+                                created_at : expect.any(String),
+                                votes : expect.any(Number),
+                                comment_count : expect.any(Number)
+                            })
+                        )
+                    })
+                    expect(articles[0].article_id).toBe(3);
+                    expect(articles[articles.length - 1].article_id).toBe(7);

```

Comment:
this test is great! 

We're missing some more testing though - we need to check that the data is coming back sorted by `created_at` in `desc` order as per the ticket instructions 

sounds like a job for Jest-Sorted to me